testing/python/fixtures.py

```diff
@@ -3339,6 +3340,33 @@ def test_foo(B):
         assert output1 == output2
 
 
+class FixtureFunctionDefTestClass:
+    def __init__(self) -> None:
+        self.i = 10
+
+    @pytest.fixture
+    def fixture_function_def_test_method(self):
+        return self.i
+
+
+@pytest.fixture
+def fixture_function_def_test_func():
```

Comment:
I would write tests that register fixtures as pytester tests, to avoid "polluting" the actual pytest-testsuite pytest session.